yeti/yurt
# Release-0.1.dev50

- ENHANCE: Add unit tests and Python 3 compatibility
  - Add tests for `yurt_core.add`
  - Add tests for `yurt_core.setup`
  - Refactor `BaseCase` out of test files
  - Cleanup unused arguments
  - Add helper method tests for `SetupTestCase`
  - Finish test for `load_orchestration_and_requirements`
  - Add test_suite configuration to `setup.py`
  - Update pycrypto to 2.6.1 (not finding issues anymore)
  - Enable `python setup.py test` invocation
  - Enable running `tox` command
  - Add entries to `.gitignore` (ie `./tox`, etc.)
  - Update requirements to include tox
  - Update `setup.py` to include tox
  - Refactor to pull gitignore.template from templates directory
  - Add first `utils.py` test: `get_vault_credentials_from_path`
  - Delete `_get_bashrc` util function
  - Fix issues with adding vault on Python 3
  - Fix issues with adding remote server on Python 3
  - Make `create_pem_file` tests pass
  - Add Python 3.5 compatibility to tests
  - Utilize `*args` to keep from having `_` in positional arguments
  - Add coverage (running `coverage run setup.py test` then `coverage report` gives coverage data)
  - Add to `.gitignore` and `travis.yml` file
  - Rename `travis.yml` to `.travis.yml`
  - Add `pointer_file` to maintain test_directory tree
  - Replace `tox` call with `python setup.py test`
  - Remove "3.4" tests
  - Refine `.coveragerc` to remove test coverage of test files
  - Fix `yurt existing` bug fix with the folder-renaming method
  - Add test to prevent `yurt existing` bug
  - Move some dependencies to dev-requirements.txt
  - Variablize some methods that call with `*args`
  - Fix `expected` calls in tests (remove uber tupling)
- Increment version number
